### PR TITLE
doc: add a note for using `configure` on Windows

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
Fixes: https://github.com/nodejs/node-gyp/issues/1149

### CODE REVIEW

**Comment by vsemozhetbyt** [comment]
I am not sure about the proper place or wording. Please, correct me.

File: README.md [hunk 1]
```diff
@@ -91,6 +91,11 @@ platform. Use `configure` for that:
 $ node-gyp configure
 ```
 
+For some Windows configurations, you may need to add `--msvs_version=2015` key:
```
Comment (bnoordhuis) [inline README.md]: I'd explain that it's only needed when auto-detection fails.

Comment (vsemozhetbyt) [inline README.md]: Is it OK?

**Comment by joaocgreis** [comment]
@vsemozhetbyt thanks for opening this PR!

I would make it something like: "Auto-detection fails for Visual C++ Build Tools 2015, so `--msvs_version=2015` needs to be added (not needed when run by npm as configured above):"

We can be clear about VCBT2015, because it is exactly and only for it that GYP fails to autodetect.

**Comment by vsemozhetbyt** [comment]
@joaocgreis Done.

**Comment by vsemozhetbyt** [comment]
ping)

**Comment by vsemozhetbyt** [comment]
@joaocgreis @bnoordhuis @gibfahn Can we land this? See https://github.com/nodejs/node-gyp/issues/1149#issuecomment-296452844

**Comment by gibfahn** [comment]
Based on https://github.com/nodejs/node-gyp/pull/1153#issuecomment-296122725, yes!

>It's no different from other repos: run CI, add commit metadata, merge. Anyone with write access can (and is welcome to) land patches.

CI just in case: https://ci.nodejs.org/view/All/job/nodegyp-test-pull-request/10/

**Comment by vsemozhetbyt** [comment]
@gibfahn But I have not write access here)

**Comment by gibfahn** [comment]
Landed in https://github.com/nodejs/node-gyp/commit/c09cf7671e6c52235784c6d3e07d9a0b1d074baa

cc/ @bnoordhuis in case I messed something up.

File: README.md [hunk 2]
```diff
@@ -91,6 +91,12 @@ platform. Use `configure` for that:
 $ node-gyp configure
 ```
 
+Auto-detection fails for Visual C++ Build Tools 2015, so `--msvs_version=2015`
+needs to be added (not needed when run by npm as configured above):
+``` bash
```
Comment (refack) [inline README.md]: you could do ```ShellSession since it's not `bash`

File: README.md [hunk 3]
```diff
@@ -91,6 +91,12 @@ platform. Use `configure` for that:
...
+$ node-gyp configure --msvs_version=2015
```
Comment (refack) [inline README.md]: maybe something like `c:\node>`

**Comment by refack** [comment]
Off I just missed the action :(

**Comment by vsemozhetbyt** [comment]
Yeah, it was a bit untidily from me, sorry.

**Comment by gibfahn** [comment]
@refack feel free to submit a PR!

**Pull request closed without merging**